src/app/services/asset.service.tsx

```diff
@@ -21,8 +21,9 @@ export class AssetService {
       if (response.status === 200) {
         return response.data;
       }
-    // tslint:disable-next-line:no-empty
-    } catch {}
+      // tslint:disable-next-line:no-empty
+    } catch {
+    }
```

Comment:
what is this ?

Reply:
say no empty catch

Reply:
@Motzart that was before me)
 I will throw the error, okay?